src/common/utxobased/engine/makeUtxoEngine.ts

```diff
@@ -268,6 +295,18 @@ export async function makeUtxoEngine(config: EngineConfig): Promise<EdgeCurrency
         coin: currencyInfo.network,
         privateKeys
       })
+      const { edgeSpendInfo } = transaction.otherParams
+      const { paymentProtocolInfo } = edgeSpendInfo.otherParams
+      if (typeof paymentProtocolInfo !== 'undefined') {
+        const publicAddress = this.getFreshAddress({}).publicAddress
```

Comment:
This 'publicAddress' is never used and can be removed.

Reply:
thank you :+1: